src/translib/transformer/xfmr_intf.go

```diff
@@ -895,6 +901,14 @@ var YangToDb_intf_subintfs_xfmr KeyXfmrYangToDb = func(inParams XfmrParams) (str
     var subintf_key string
     var err error
 
+    pathInfo := NewPathInfo(inParams.uri)
+    idx := pathInfo.Var("index")
+    if idx != "0" {
+        errStr := "Not supported Index: " + idx
```

Comment:
Let's change it to "Invalid Interface Index" instead of "Not Supported".

Reply:
Done.

Reply:
Hey Justin, Its not interface index, it will add more confusion with if index, better use sub interface index.

Reply:
You are right @rvasanthm , I didn't think when I got the previous comment. It will bring in more confusion. I will change it.